instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/logs/GdprLogsE2ETests.kt

```diff
@@ -0,0 +1,201 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.logs
+
+import androidx.test.platform.app.InstrumentationRegistry
+import com.datadog.android.Datadog
+import com.datadog.android.log.Logger
+import com.datadog.android.nightly.utils.defaultTestAttributes
+import com.datadog.android.nightly.utils.initializeSdk
+import com.datadog.android.nightly.utils.measureLoggerInitialize
+import com.datadog.android.nightly.utils.measureSdkInitialize
+import com.datadog.android.nightly.utils.measureSetTrackingConsent
+import com.datadog.android.nightly.utils.stopSdk
+import com.datadog.android.privacy.TrackingConsent
+import com.datadog.tools.unit.forge.aThrowable
+import fr.xgouchet.elmyr.junit4.ForgeRule
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class GdprLogsE2ETests {
+
+    lateinit var logger: Logger
+
+    @get:Rule
+    val forge = ForgeRule()
+
+    @Before
+    fun setup() {
+        stopSdk()
+    }
+
+    // region Tests
+
+    /**
+     * apiMethodSignature: Credentials#constructor(String, String, String, String?, String? = null)
+     */
+    @Test
+    fun logs_credentials_consent_pending() {
```

Comment:
probably method name should be updated to match `testMethodName`? here and below.

also what we are testing here: sdk initialization + logs initialization?